Extension/src/LanguageServer/configurations.ts

```diff
@@ -586,8 +586,8 @@ export class CppProperties {
         this.compileCommandFileWatchers = []; //reset it
         let filePaths: Set<string> = new Set<string>();
         this.configurationJson.configurations.forEach(c => {
-            if (c.compileCommands !== undefined && fs.existsSync(c.compileCommands)) {
-                filePaths.add(c.compileCommands);
+            if (c.compileCommands !== undefined && fs.existsSync(this.resolvePath(c.compileCommands, os.platform() === "win32"))) {
+                filePaths.add(this.resolvePath(c.compileCommands, os.platform() === "win32"));
```

Comment:
Save the result of `this.resolvePath(c.compileCommands, os.platform() === "win32")` and then use in fs.existsSync(...) and filePaths.add(...)? This way, it does not have to go resolvePath twice.

Reply:
you're right, changed thanks